src/server/external-services/pathway-commons.js

```diff
@@ -17,74 +17,87 @@ const fetchOptions = {
   }
 };
 
-const _sanitize = (s) => {
+//Pathway Commons HTTP GET request; options.cmd = 'pc2/get', 'pc2/search', 'pc2/traverse', 'pc2/graph', etc.
+let query = opts => {
+  let queryOpts = _.assign( { user: 'app-ui', cmd: 'pc2/get' }, opts);
```

Comment:
Might make sense to add these pc user opts to the top of the scope in this module.